native/packages/react-native-bpk-component-text-input/src/styles.js

```diff
@@ -28,22 +28,22 @@ import {
   fontFamily,
   spacingMd,
   spacingSm,
+  lineHeightSm,
+  lineHeightBase,
   textBaseFontSize,
   textBaseFontWeight,
-  textBaseLineHeight,
```

Comment:
These changes are interesting. Does it still look resonable with this? I guess what we really wanna do here is use the calculated line heights not the tokens

Reply:
The tokens evaluate to the same values, so it ends up looking the same.

E.g.:

```
export const lineHeightXs = 18;
export const textXsLineHeight = 18;
```

I don't think it's a problem to use the tokens here though - we use them in other components too, e.g. in `BpkButton`. 

I figured since the card for this is just to remove the text ones, I just replaced this component to no longer use those in favour of an alternative that other components are also using.

Reply:
> The tokens evaluate to the same values, so it ends up looking the same.

Does it though? The tokens evaluate to the same value, but since we are no longer setting the line heights the actual line heights will be different. For example what happnes in the precense of font scaling on Android?